### DISCUSSION TITLE
Bugs in programs

### DISCUSSION THREAD

**Discussion by OnTheWallaby** [community/community#23040] [New to GitHub]
<p>Where or how do you report and give feedback on issues with a program?</p>
<p>This is the program I am trying to use, and fails completely:<br>
<a href="https://github.com/k3b/ToGoZip" rel="nofollow noopener">https://github.com/k3b/ToGoZip</a><br>
Looks great and appears to have good integration into Android.  Sadly fails and crashes.</p>
<p>I did try and look around to report issues, but couldn’t find anything.</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/onthewallaby">@onthewallaby</a>,</p>
<p>To report bugs or give feedback on a project you can <a href="https://help.github.com/en/articles/creating-an-issue" rel="nofollow noopener">create a new issue</a> in the repository for the program. Follow the link below to begin creating a new issue for the program in question.</p>
<p><a href="https://github.com/k3b/ToGoZip/issues/new" rel="nofollow noopener">https://github.com/k3b/ToGoZip/issues/new</a></p>
<p>Make sure you try to provide as much details as possible when reporting bugs or providing feedback.</p>

**Reply by Carib909** [reply]
I did as asked but the Submit button is greyed out. Eleven Clock does not start after install and I cannot provide logs because right clicking does nothing but ask to remove it from Task tray or to click to start the app. Clicking to start the app does nothing.
